stdpopsim/drosophila_melanogaster.py

```diff
@@ -75,13 +75,44 @@ class Comeron2012_dm6(genetic_maps.GeneticMap):
 
 class SheehanSongThreeEpoch(models.Model):
     """
-    The three epoch model estimated for African samples from Sheehan and Song
-
-    .. todo:: document this model, including the original publications
-        and clear information about what the different population indexes
-        mean.
-
-    """
+    Model Name:
+        SheehanSongThreeEpoch
+    
+    Model Description:
+        The three epoch (modern, bottleneck, ancestral) model estimated for a
+        single African Drosophila Melanogaster population from `Sheehan and Song <https:/
+        /doi.org/10.1371/journal.pcbi.1004845>`_ . Population sizes are estimated by a 
+        deep learning model trained on simulation data. NOTE: Due to differences in 
+        coalescence units between PSMC (2N) and msms (4N) generation times were doubled 
```

Comment:
Does this last note help? Everything is in units of generations here, so the scaling used in the original publication doesn't really enter into it, does it?

Reply:
so unfortunately the authors seem to have had their own factor of two error and we should probably document it....